packages/typescript-estree/package.json

```diff
@@ -61,5 +61,8 @@
     "glob": "^7.1.4",
     "lodash.isplainobject": "4.0.6",
     "typescript": "*"
+  },
+  "optionalDependencies": {
```

Comment:
ditto for needing this in `eslint-plugin` - might as well be consistent :)
parser consumes typescript-estree so it isn't needed there.